src/pantry.js

```diff
@@ -2,6 +2,18 @@ class Pantry {
   constructor(userIngredients) {
     this.contents = userIngredients;
   }
+
+  calculateIngredients() {
+
+  }
+
+  calculateMissingIngredients() {
+
+  }
+
+  removeIngredients() {
```

Comment:
any reason why these methods are empty?

Reply:
Oh no I forgot to remove this after we decided to focus on iteration 1's classes. This has since been updated! Great eye!

Reply:
ah no worries! I was just curious :D